### ISSUE TITLE
Head and p tags closing incorrectly

### ISSUE THREAD

**Issue by fishb6nes** [lukasjapan/koreander#6]
Heya Lukas,
The Japanese beers example from the readme seems to be closing the `<head>` tag incorrectly.
Rather than:
```
<!DOCTYPE html>
<html>
    <head>
        ...
    </head>
    <body>
        ...
    </body>
</html>
```

The generated HTML looks like (inspect the payload, not the HTML inspector in the browser):
```
<!DOCTYPE html>
<html>
    <head>
        ...
    <body>
        ...
    </body>
    </head>
</html>
```

I've observed a similar issue with `%p` tags that include a linebreak. For example:
```
!!! 5
%html
    %body
        %p
            line 1
            line 2
            line 3
        %div
```

Renders correctly, but introducing a linebreak in the paragraph:
```
!!! 5
%html
    %body
        %p
            line 1
            %br
            line 2
        %div
```

Instead results in:
```
<!DOCTYPE html>
<html>
    <body>
        <p>
            line 1
            <br>
            line 2
        <div></div>
        </p>
    </body>
</html>
```

Am I nesting the linebreak incorrectly in the paragraph or are there some inconsistencies in how the indentations are read?

Many thanks,
fish

**Comment by lukasjapan** [comment]
Ouch, seems like invalid handling of a closing tag, if there is no closing tag.

Will look into that soon.

**Comment by fishb6nes** [comment]
Thank you, there's no rush

**Comment by lukasjapan** [comment]
fixed in `0.0.7`
